Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "pretty_env_logger"
-version = "0.4.0" # don't forget to update html_root_url
+version = "0.4.1" # don't forget to update html_root_url
```

```suggestion
version = "0.4.0" # don't forget to update html_root_url
```

I'd recommend leaving the version bump off:

1. This is likely a breaking change, since `env_logger` is publicly exposed.
2. I prefer version changes to be a standalone commit that can be tagged as "the release".

---

> This is likely a breaking change, since env_logger is publicly exposed.

I must have missed where it was exposed publicly, sorry about that. The only crate I saw in the API on docs.rs was `log`.

> I prefer version changes to be a standalone commit that can be tagged as "the release".

Thanks for the info, I'll keep that in mind :thumbsup: